.github/workflows/mac-artifact.yml

```
@@ -12,7 +12,28 @@ on:
         type: string
 jobs:
   build:
-    runs-on: macos-11
+    strategy:
+      matrix:
+        os: [macos-11, macos-14]
+        include:
+          - os: macos-11
```

```suggestion
        os: [macos-12, macos-14]
        include:
          - os: macos-12
```
macos-11 runners are no longer supported.